src/drivers/mct/cime_config/config_component_cesm.xml

```diff
@@ -221,7 +221,7 @@
       <value compset="_CAM\d+%WCXI">288</value>
       <value compset="_DATM%COPYALL_NPS">72</value>
       <value compset="_DATM.*_CLM">48</value>
-      <value compset="_DATM.*_DICE.*_POP2">4</value>
+      <value compset="_DATM.*_DICE.*_POP2">24</value>
```

Comment:
@mvertens is this change okay for mct?

Reply:
no - can you please revert this to its original values. This was introduced in my testing for the nuopc branch and talking to Gustavo and Alper it seems that they want the original values kept. Thanks for checking.